diff --git a/contracts/AvocadoMultisig/lib/Initializable.sol b/contracts/AvocadoMultisig/lib/AvocadoMultisigInitializable.sol
index 42ec869..7e928db 100644
--- a/contracts/AvocadoMultisig/lib/Initializable.sol
+++ b/contracts/AvocadoMultisig/lib/AvocadoMultisigInitializable.sol
@@ -1,9 +1,13 @@
 // SPDX-License-Identifier: MIT
-// OpenZeppelin Contracts (last updated v4.8.1) (proxy/utils/Initializable.sol)
+pragma solidity >=0.8.18;
 
-pragma solidity ^0.8.2;
+import { Address } from "@openzeppelin/contracts/utils/Address.sol";
 
-import "../../utils/Address.sol";
+import { AvocadoMultisigVariables } from "../AvocadoMultisigVariables.sol";
+
+/// @dev contract copied from OpenZeppelin Initializable but with storage vars moved to AvocadoMultisigVariables.sol
+/// from OpenZeppelin Contracts (last updated v4.8.1) (proxy/utils/Initializable.sol)
+/// see https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v4.8.1/contracts/proxy/utils/Initializable.sol
 
 /**
  * @dev This is a base contract to aid in writing upgradeable contracts, or any kind of contract that will be deployed
@@ -54,17 +58,17 @@ import "../../utils/Address.sol";
  * ```
  * ====
  */
-abstract contract Initializable {
+abstract contract AvocadoMultisigInitializable is AvocadoMultisigVariables {
     /**
      * @dev Indicates that the contract has been initialized.
      * @custom:oz-retyped-from bool
      */
-    uint8 private _initialized;
+    // uint8 private _initialized; // -> in AvocadoMultisigVariables
 
     /**
      * @dev Indicates that the contract is in the process of being initialized.
      */
-    bool private _initializing;
+    // bool private _initializing; // -> in AvocadoMultisigVariables
 
     /**
      * @dev Triggered when the contract has been initialized or reinitialized.
